demos/fractal/fractal.cpp

```diff
@@ -149,16 +149,16 @@ MyCanvas::MyCanvas(wxFrame *frame):
  wxWindow(frame, wxID_ANY)
 {
     wxColour wxCol1(255,255,255);
-    SnowPen = wxPen(wxCol1, 2, wxSOLID);
+    SnowPen = wxPen(wxCol1, 2, wxPENSTYLE_SOLID);
```

Comment:
Solid style is also the default here and could (and should IMO) be just removed.